lib/paramedic.js

```diff
@@ -377,42 +381,37 @@ class ParamedicRunner {
     }
 
     getCommandForStartingTests () {
-        let cmd = this.config.getCli() + ' ' + this.config.getAction() + ' ' + this.config.getPlatformId() + utilities.PARAMEDIC_COMMON_CLI_ARGS;
-
-        const addConfigArgs = (cmd) => {
-            if (this.config.getArgs()) {
-                cmd += ' ' + this.config.getArgs();
-            }
-
-            return cmd;
-        };
-
-        if (this.config.getPlatformId() === utilities.BROWSER) {
-            return addConfigArgs(cmd);
-        }
-
-        const paramedicTargetChooser = new ParamedicTargetChooser(this.tempFolder.name, this.config);
-
-        if (this.config.getAction() === 'build' || (this.config.getPlatformId() === utilities.WINDOWS && this.config.getArgs().indexOf('appx=8.1-phone') < 0)) {
+        let cmd = [
+            this.config.getCli(),
+            this.config.getAction(),
+            this.config.getPlatformId()
+        ]
+            .concat(utilities.PARAMEDIC_COMMON_ARGS)
+            .concat(this.isBrowser ? [this.config.getArgs()] : []);
+
+        if (this.isBrowser) {
+            return cmd.join(' ');
+        } else if (this.config.getAction() === 'build' || (this.isWindows && this.config.getArgs().indexOf('appx=8.1-phone') < 0)) {
             // The app is to be run as a store app or just build. So no need to choose a target.
-            return Q(addConfigArgs(cmd));
+            return Q(cmd.join(' '));
```

Comment:
this isn't new from you, but wtf is it wrapped in `Q` here but not for browser https://github.com/apache/cordova-paramedic/pull/159/files#diff-ff902472818c9b031ca73ec778f68526R393 ?

Reply:
that being said: this line here is probably dead code anyway that is not used at all because windows phone 8.1 is not present any more.

Reply:
I would have to dig into this but I think the `Q()` wrapping is actually correct. But why the browser case does not need it, I do not know yet.

Why I feel the `Q` wrapping is correct is because the last return is also wraps and I know it is hit for the iOS local tests.